pkg/apis/airflow/v1alpha1/airflowcluster_types.go

```diff
@@ -534,6 +642,23 @@ func (b *AirflowCluster) OwnerRef() *metav1.OwnerReference {
 	})
 }
 
+// Finalize finalizes AirflowCluster component when it is deleted
+func (s *AirflowCluster) Finalize(rsrc interface{}, observed, dependent *resource.Bag) error {
```

Comment:
you could remove this As redis finalizer is more targetted.

Reply:
I've already removed for `AirflowCluster`, and also for `AirflowBase` as I added two.